scanner/src/main/kotlin/CommandLinePathScannerWrapper.kt

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2022 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.scanner
+
+import java.io.File
+import java.io.IOException
+
+import kotlin.time.measureTimedValue
+
+import org.apache.logging.log4j.kotlin.Logging
+
+import org.ossreviewtoolkit.model.ScannerDetails
+import org.ossreviewtoolkit.scanner.experimental.PathScannerWrapper
+import org.ossreviewtoolkit.utils.common.CommandLineTool
+import org.ossreviewtoolkit.utils.common.Os
+
+/**
+ * A [PathScannerWrapper] that is executed as a [CommandLineTool] on the local machine.
+ */
+abstract class CommandLinePathScannerWrapper(
```

Comment:
Nit: Why not just "CommandLineScannerWrapper"?

Reply:
I guess to be more explicit. I'm fine with either name but if it's renamed I'd prefer to do this after this PR is merged.

Reply:
I'm also ok with sticking to the longer name. I was just curious whether you'd like to emphasize that there might be "CommandLine" scanners that are not "PathScanners".